src/main/java/edu/ucar/unidata/rosetta/repository/JdbcPropertiesDao.java

```diff
@@ -0,0 +1,63 @@
+package edu.ucar.unidata.rosetta.repository;
+
+import edu.ucar.unidata.rosetta.domain.RosettaProperties;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+
+import org.springframework.dao.DataRetrievalFailureException;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.jdbc.core.support.JdbcDaoSupport;
+
+/**
+ * @author oxelson@ucar.edu
+ */
+public class JdbcPropertiesDao extends JdbcDaoSupport implements PropertiesDao {
+
+    protected static Logger logger = Logger.getLogger(JdbcPropertiesDao.class);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') "logger" is the name of a field in "DaoSupport". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2387)